src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static Random random = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
We should also avoid static methods and variables, the same for other classes + check common mistakes for more details.
And I wanted to mention that it is not the guided way to change the return type and method signature, but usually, it is a proper way to store Data in enum type instead of string in `Ball` class, so we can leave it as it is